package.json

```diff
@@ -199,10 +199,12 @@
       "v6",
       {
         "name": "v6-beta",
+        "channel": "beta",
         "prerelease": "beta"
```

Comment:
Is this still needed?

Reply:
I believe it is. "prerelease" will be used as the suffix by `semantic-release` and generate releases like `6.0.0-beta`. Otherwise, this will make releases from `v6-beta` without the suffix.

Documentation: https://semantic-release.gitbook.io/semantic-release/usage/workflow-configuration#prerelease

Reply:
Ah yes, that makes sense. Thanks!

Reply:
V6 Beta can be removed completely